pulp_rpm/app/migrations/0026_checksum_null.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.16 on 2020-10-09 14:58
+
+from django.db import migrations, models
+from django.db.migrations import exceptions
+
+
+def check_checksum(apps, schema_editor):
+    checksum_model = apps.get_model("rpm", "Checksum")
+    q_bad_dist_tree = checksum_model.objects.filter(checksum=None)
+    if q_bad_dist_tree:
+        dist_tree_pks = {
+            c.distribution_tree.pk for c in checksum_model.objects.filter(checksum=None)
```

Comment:
Is there a way to provide users with hrefs? I think it's more user friendly, they might not know what to do with a list of IDs.

Reply:
I think hardcoded hrefs will save resources instead of getting each objects pulp_href.